crates/transaction-emitter-lib/src/emitter/account_minter.rs

```diff
@@ -70,48 +70,67 @@ impl<'t> AccountMinter<'t> {
         total_requested_accounts: usize,
     ) -> Result<Vec<LocalAccount>> {
         let mut accounts = vec![];
-        let expected_num_seed_accounts =
-            if total_requested_accounts / req.rest_clients.len() > MAX_CHILD_VASP_NUM {
```

Comment:
If we are getting rid of the VASP feature, let's also remove corresponding `reuse_accounts` argument in `ClusterArgs`